frontend/constants/colors.ts

```diff
@@ -18,4 +18,8 @@ export const COLOR = {
   GRAY_2: '#4D596A',
   SUCCESS: '#52C41A',
   WARNING: '#FA8C16',
+
+  // v1 colors
+  V1_SUCCESS: '#28B858',
+  V1_GRAY_1: '#EDF2F7',
```

Comment:
tiny, buy do you think we can simply update values for the existing SUCCESS and GRAY_1? otherwise we'll need to get back cleaning this up